File: BaekJoon_6Level.py
# 10818번
n = int(input())
arr = list(map(int, input().split()))
print(min(arr), max(arr)) # max, min : 최대, 최소값 구하는 함수

# 2562번
n = []
for i in range(9):
    n.append(int(input()))
max = n[0]
cnt = 0
for i in range(9):
    if max <= n[i]:
        max = n[i]
        cnt = i + 1
print(max)
print(cnt)

# 2577번
a = int(input())
b = int(input())
c = int(input())
mul = list(str(a * b * c))

for i in range(10):
    print(mul.count(str(i)))

# 3052번
n = []
for i in range(10):
    _n = int(input())
    n.append(_n % 42)
n = set(n) # set : 중복을 제거한 집합 자료형
print(len(n))

# 1546번
n = int(input())
arr = list(map(int, input().split()))
m = max(arr)
r_arr = []
for i in range(n):
    r_arr.append(arr[i] / m * 100)
print(format(sum(r_arr) / n, ".6f"))

# 8958번
n = int(input())
test = []
score = []
for i in range(n):
    test.append(str(input()))
    score.append(0)
for i in range(n):
    cnt = 0
    for j in range(len(test[i])):
        if test[i][j] == 'O':
            cnt += 1
            score[i] += cnt
        else:
            cnt = 0
for i in range(n):
    print(score[i])

# 4344번
c = int(input())
per_cnt = []
for i in range(c):
    score = list(map(int, input().split()))
    avg = (sum(score) - score[0]) / score[0]
    cnt = 0
    for j in range(1, len(score)):
        if score[j] > avg:
            cnt += 1
    print('%0.3f' % round(cnt / score[0] * 100, 3), '%', sep='')
    # round : 반올림(반올림할 수, 표시할 자릿 수)

File: BaekJoon_6Level.cpp
#include<iostream>
#include<string>

////10818번
//int main() {
//	int n;
//	std::cin >> n;
//	int* arr = new int[n];
//	for (int i = 0; i < n; i++)
//		std::cin >> arr[i];
//	int max = arr[0];
//	int min = arr[0];
//	for (int i = 0; i < n; i++) {
//		if (max < arr[i])
//			max = arr[i];
//		if (min > arr[i])
//			min = arr[i];
//	}
//	std::cout << min << " " << max;
//	delete[] arr;
//}

////2562번
//int main() {
//	int score[9], cnt = 1;
//	for (int i = 0; i < 9; i++)
//		std::cin >> score[i];
//	int max = score[0];
//	for (int i = 0; i < 9; i++) {
//		if (max < score[i]) {
//			max = score[i];
//			cnt = i + 1;
//		}
//	}
//	std::cout << max << '\n' << cnt;
//}

////2577번
//int main() {
//	int a, b, c, cnt[10] = { 0 }, chk = 0;
//	std::cin >> a >> b >> c;
//	std::string mul = (std::to_string)(a * b * c);
//	for (int i = 0; i < mul.length(); i++) {
//		cnt[mul[i] - 48]++;
//	}
//	for (int i = 0; i < 10; i++)
//		std::cout << cnt[i] << '\n';
//}

////3052번
//int main() {
//	int a, rest[10], cnt = 1;
//	bool chk[42] = { false };
//	for (int i = 0; i < 10; i++) {
//		std::cin >> a;
//		rest[i] = a % 42;
//	}
//	for (int i = 1; i < 10; i++)
//		if (rest[0] != rest[i])
//			if (chk[rest[i]] == false) {
//				chk[rest[i]] = true;
//				cnt++;
//			}
//	std::cout << cnt;
//}

////1546번
//int main() {
//	int n;
//	double max = 0, sum = 0, avg = 0;
//
//	std::cin >> n;
//	double* arr = new double[n];
//	for (int i = 0; i < n; i++) {
//		std::cin >> arr[i];
//	}
//	for (int i = 0; i < n; i++) {
//
//		if (max < arr[i]) {
//			max = arr[i];
//		}
//
//	}
//	for (int i = 0; i < n; i++) {
//		arr[i] = (arr[i] / max * 100.0);
//		sum += arr[i];
//	}
//	std::cout << std::fixed;
//	std::cout.precision(2);
//	std::cout << sum / n << std::endl;
//	delete[] arr;
//}

////8958번
//int main() {
//	int n, cnt = 0, score = 0;
//	std::cin >> n;
//	std::string* str = new std::string[n];
//	int* s_list = new int[n];
//	for (int i = 0; i < n; i++) {
//		std::cin >> str[i];
//	}
//	for (int i = 0; i < n; i++) {
//		score = 0;
//		cnt = 0;
//		for (int j = 0; j < str[i].length(); j++) {
//			if (str[i][j] == 'O') {
//				if (j >= 1 && str[i][j - 1] == 'O') {
//					cnt += 1;
//					score += cnt;
//				}
//				else {
//					cnt = 1;
//					score += cnt;
//				}
//			}
//		}
//		s_list[i] = score;
//	}
//	
//	for(int i=0;i<n;i++)
//		std::cout << s_list[i] << '\n';
//	delete[] s_list, str;
//}

//4344번
int main() {
	int c, n, sum = 0, cnt = 0;
	int score[1000];
	double avg, rest;

	std::cin >> c;
	for (int i = 0; i < c; i++)
	{
		rest = 0;
		cnt = 0;
		sum = 0;
		std::cin >> n;
		for (int j = 0; j < n; j++)
		{
			std::cin >> score[j];
			sum += score[j];
		}
		avg = (double)sum / n;
		for (int j = 0; j < n; j++)
		{
			if (avg < score[j])
				cnt++;
		}
		rest = (double)cnt / n * 100;

		std::cout << std::fixed;
		std::cout.precision(3);
		std::cout << rest << "%\n";
	}
}